### ISSUE TITLE
Не работают расширенные обработчики элементов формы с директивой После при загрузке конфигурации из EDT в базу

### ISSUE THREAD

**Issue by lexandersergeev** [1C-Company/1c-edt-issues#1517]
### Описание ошибки

Если в расширении расширить обрабочик события элемента формы, например события Выбор таблицы формы, то после обновления конфигурации из EDT расширенный обработчик не работает.

### Как воспроизвести

Добавьте в расширение любую форму с таблицей.
Создайте расширенный обработчик события `Выбор` таблицы на форме с директивой `После`.
Обновите конфигурацию базы из EDT.
В EDT связь обработчика есть, а в конфигураторе нет.
Если сделать экспорт расширения в XML-формат платформы и попробовать его загрузить, конфигуратор сообщает ошибку: `Файл - ..\Form.xml: Событие "Selection" элемента управляемой формы "Item.Таблица" не было загружено т.к. не является событием данного элемента.`

### Скриншоты

![image](https://github.com/user-attachments/assets/984496fc-c608-4163-94a8-26d6893ef91c)
![image](https://github.com/user-attachments/assets/a0115b16-982d-45a9-b597-2c89e735f4a4)

### Ожидаемое поведение

Связь расширенного обработчика сохраняется при обновлении конфигурации, и расширение обработчика работает.

### Лог рабочей области

[logs.zip](https://github.com/user-attachments/files/17036674/logs.zip)

### Версия 1С:EDT

Ruby 2023.3.6

### Операционная система

Windows

### Версия платформы 1С:Предприятие 8

8.3.24.1691

**Comment by Jimmo910** [comment]
раньше тоже было терялась эта связь. Из-за чего я лично стараюсь не расширять события, а расширять только методы. правда Метода может не быть и тогда попадос...

**Comment by vadimeg** [comment]
См. https://github.com/1C-Company/1c-edt-issues/issues/1285

**Comment by lexandersergeev** [comment]
> См. #1285
8.3.25 ещё не пробовал. Изначально не смог найти похожий ишью. Получается, в платформу регресс завезли? На 8.3.24.1691 воспроизводится.

**Comment by vadimeg** [comment]
> 8.3.25 ещё не пробовал. Изначально не смог найти похожий ишью. Получается, в платформу регресс завезли? На 8.3.24.1691 воспроизводится.

Да, регрессия, просто исправили не для всех случаев, как выяснилось. В 25-й та же проблема. В 23-й все ОК.

**Comment by nikolay-martynov** [comment]
Здравствуйте. Приношу извинения за возникшую проблему и за то, что она не была исправлена в прошлый раз до конца. К сожалению, необходимо подождать публикации исправительной сборки платформы для 70093565. Я дополнительно сообщу, когда исправление будет доступно.

**Comment by nikolay-martynov** [comment]
Данная проблема была исправлена в платформе 1С:Предприятие и требуется её обновление до версий 8.3.24.1731, 8.3.25.1447 или выше. В EDT требования к версии платформы 1С:Предприятие будут повышены соответствующим образом в следующем выпуске 2024.2.

**Comment by nikolay-martynov** [comment]
К сожалению, в автономном сервере проблема исправлена не до конца. При необходимости использования расширенных обработчиков пожалуйста используйте приложение на информационной базе, а не на автономном сервере. О доступности исправительной сборки платформы 1С:Предприятие сообщу дополнительно.

**Comment by Jimmo910** [comment]
8.3.25.1496 в тесте появилась
@nikolay-martynov в ней не исправлена проблема?

**Comment by nikolay-martynov** [comment]
@Jimmo910 К сожалению, не все сценарии ещё исправлены. Мы дополнительно сообщим версию платформы 1С:Предприятие, где проблема полностью исправлена.

**Comment by nikolay-martynov** [comment]
@Jimmo910 Здравствуйте. Проблема частично исправлена в 1С:Предприятие 8.3.25.1526. Для полного исправления требуется использовать версии 8.3.26.1515, 8.3.27.1314 или выше. В EDT 2024.2 требования к версиям платформы будут повышены соответствующим образом.

**Comment by Dmitriy83** [comment]
@nikolay-martynov Здравствуйте! Кажется, проблема так и сохранилась. Проверили на EDT версии Ruby 2024.2.4. Платформа 1С 8.3.26.1540. В ЕДТ обработчики есть (см. скрин). Но после загрузки в базу в Конфигураторе они отсутствуют. Следует ли открыть новую задачу?

![Image](https://github.com/user-attachments/assets/800d4c48-add8-4f20-a416-5d2df4959b98)

![Image](https://github.com/user-attachments/assets/c7fde908-2cff-40fe-a8a7-1be1204f9f6d)
